skbio/stats/distance/tests/test_mantel.py

```diff
@@ -549,7 +552,7 @@ def test_minimal_compatible_input(self):
         # input as array_like
         obs = pwmantel((self.minx, self.miny, self.minz),
                        alternative='greater')
-        assert_data_frame_almost_equal(obs, self.exp_results_minimal)
+        assert_data_frame_almost_equal(obs, self.exp_results_minimal) #
```

Comment:
What does the trailing `#` do?

Reply:
Accidentally left it there, it's been removed with the most recent commits.